compiler/rustc_const_eval/src/util/alignment.rs

```diff
@@ -3,22 +3,21 @@ use rustc_middle::mir::*;
 use rustc_middle::ty::{self, TyCtxt};
 use tracing::debug;
 
-/// Returns `true` if this place is allowed to be less aligned
-/// than its containing struct (because it is within a packed
-/// struct).
+/// Returns the packed alignment if this place is allowed to be less aligned
+/// than its type normally requires (because it is within a packed struct).
 pub fn is_disaligned<'tcx, L>(
```

Comment:
Nit: The `is_` part is kinda wrong now, maybe the function should be renamed?

Maybe something like `misalignment`? (would also fix my problem with the use of "disaligned" here which sounds weird, the proper term is "misaligned").